### ISSUE TITLE
Python 3 issue in splice.py

### ISSUE THREAD

**Issue by tpetazzoni** [giampaolo/pyftpdlib#381]
The tarball of pyftpdlib in version 1.5.0 available on Pypi at https://pypi.python.org/packages/source/p/pyftpdlib/pyftpdlib-1.5.0.tar.gz includes a file called splice.py that doesn't byte-compile properly with Python 3. I wanted to send a pull request to fix this, but this file doesn't exist in the Github repository, and so I'm rather confused as to why the Pypi tarball has it.

In any case, here is below the fix that needs to be applied on splice.py to make it build with Python 3:

```
Index: b/pyftpdlib/splice.py
===================================================================
--- a/pyftpdlib/splice.py
+++ b/pyftpdlib/splice.py
@@ -125,7 +125,7 @@
     # Single accept, we'll clean up once this one connection has been handled.
     # Yes, this is a very stupid server indeed.
     conn, addr = sock.accept()
-    print 'Connection from:', addr
+    print('Connection from:', addr)
 
     # Set up some subprocess which produces some output which should be
     # transferred to the client.
@@ -138,7 +138,7 @@
     # We need the integer FDs for splice to work
     pipe_fd = proc.stdout.fileno()
     conn_fd = conn.fileno() #pylint: disable-msg=E1101
-    print 'Will splice data from FD', pipe_fd, 'to', conn_fd
+    print('Will splice data from FD', pipe_fd, 'to', conn_fd)
 
     transferred = 0
 
@@ -162,7 +162,7 @@
 
         transferred += done
 
-    print 'Bytes transferred:', transferred
+    print('Bytes transferred:', transferred)
 
     # Close client and server socket
     conn.close()
```

**Comment by giampaolo** [comment]
Ahhhh that's a bummer. Not sure how splice.py ended up there. It was an experiment I was doing. I will release a new version soon.

**Comment by giampaolo** [comment]
OK I have just released a new 1.5.1 version
